docs/refresh-token.wsd

```
@@ -0,0 +1,42 @@
+@startuml
+    autonumber
+    title "Refresh token sequence diagram"
+
+    actor client as "User Logged"
+    participant bff as "BFF"
+    box "Protected stack" #LightCyan
+    participant user as "User"
+    end box
+    database db as "Database"
+
+    client -> bff: "Refresh token"
+    activate bff
+    bff -> user: "Send refresh token"
+    activate user
+    user-> db: "Search  refresh token"
+    activate db
+    db --> user: "Return refresh token"
+    deactivate db
+    alt Refresh token is valid
+        user -> user: "Create new access token"
+        user -> db: "Save new access token"
+        activate db
+        deactivate db
+        user --> bff: "Authorized [New Access Token]"
+        bff --> client: "Authorized [New Access Token]"
+        else Refresh token is expired
```

If the refresh token is not Ok, should just return an error. Do not generate an valid token to an expired refresh token.